extensions/standard-processors/tests/unit/ReplaceTextTests.cpp

```diff
@@ -93,7 +93,7 @@ TEST_CASE("Regex Replace works correctly in ReplaceText", "[applyReplacements][R
   replace_text.setSearchRegex("a\\w+e");
   replace_text.setReplacementValue("orange");
 
-  CHECK(replace_text.applyReplacements("") == "");
```

Comment:
Maybe it would be better to create a NOLINT block for this issue with `// NOLINTBEGIN(readability-container-size-empty)`